snappi_ixnetwork/trafficitem.py

```diff
@@ -203,35 +468,55 @@ def config(self):
         - CREATE TrafficItem for any config.flows[*].name that does not exist
         - UPDATE TrafficItem for any config.flows[*].name that exists
         """
-        ixn_traffic_item = self._api._traffic_item
-        self.flows_has_latency = []
-        self.flows_has_timestamp = []
-        self.flows_has_loss = []
-        self.latency_mode = None
-        self._api._remove(ixn_traffic_item, self._api.snappi_config.flows)
-        if len(self._api.snappi_config.flows) > 0:
-            for flow in self._api.snappi_config.flows:
-                self._endpoint_validation(flow)
-                args = {
-                    "Name": flow.name,
-                    "TrafficItemType": "l2L3",
-                    "TrafficType": self._get_traffic_type(flow),
-                    "SrcDestMesh": self._get_mesh_type(flow),
-                }
-                ixn_traffic_item.find(
-                    Name="^%s$" % self._api.special_char(flow.name),
-                    TrafficType=args["TrafficType"],
+        self.remove_ixn_traffic()
+        if len(self._api._config.flows) == 0:
+            return
+        with Timer(self._api, "Flows configuration"):
+            ixn_traffic_item = self.get_ixn_config(self._api._config)[0]
+            self.flows_has_latency = []
+            self.flows_has_timestamp = []
+            self.flows_has_loss = []
+            self.latency_mode = None
+            if ixn_traffic_item.get("trafficItem") is None:
+                # TODO raise Exception
+                return
+            ixn_traffic_item = ixn_traffic_item.get("trafficItem")
+            tr_json = {"traffic": {"xpath": "/traffic", "trafficItem": []}}
+            # with Timer(self._api, "json processing for traffic"):
+            for i, flow in enumerate(self._api._config.flows):
+                tr_item = {"xpath": ixn_traffic_item[i]["xpath"]}
```

Comment:
Traffic Items are accessing according to the order. So access through index ("i") probably do not have any issue. I feel it will better to access that using "name" rather than index ("i")

Reply:
as this approach is, remove traffic then create. getting the flows with name will add one more code block like validating the name. which would be a over head